docs/source/using/experimental-features.adoc

```diff
@@ -73,6 +73,11 @@ If you prefer to use dynamic assignment, you can use:
 $ minishift ip --set-dhcp
 ----
 
+[NOTE]
+====
+Use `minishift config set static-ip true` to set static ip automatic for supported hypervsiors.
```

Comment:
Just rephrasing: `to automatically set the ip as static during start, for supported hypervisors.`

Or just change automatic => automatically.